### ISSUE TITLE
Server returns 'test value firstName' and lots of other issues. Is the Cosync service still being maintained?

### ISSUE THREAD

**Issue by agape-apps** [Cosync/CosyncSamples#6]
Hi Richard Krueger, I just tried the sample React native app. I really like theCosync  service as it was easy to get started and would like to use it for my app. I found Cosync through one of your articles on Medium.

But there are quite a few issues with the app and the service, therefore first of all:

_Is the Cosync service still being maintained? Will the service still be available in 12 months?_

One reason I ask, is because the certificate of the https://cosync.io domain is expired. Also the React Native package https://www.npmjs.com/package/cosync-jwt-react-native only gets about one download per week, showing very slow adoption. Another reason is, because the [change-log 'blog' ](https://cosync.net/changelog/)has not been updated for about 10 months.

I will bring up just this issue now which I had with the demo app:  The `global.userData.data` which is metaData presumably returned from the server all shows `test value ...` instead of the name:

```
global.userData.data 
{"status":"active","createdAt":"2021-07-30T07:21:39.000Z","updatedAt":"2021-07-30T07:21:39.000Z",
"handle":"user-bbb@email.com",
"twoFactorPhoneVerification":false,"twoFactorGoogleVerification":false,
"appId":"143b39f20f3643019ee60bf5b353cc6a",

"metaData":{"user_data":{"name":{"first":"test value firstName","last":"test value lastName"},"email":"test value email"}},

"lastLogin":"2021-07-30T07:21:54.000Z"}
```
from the console log

**Comment by rckrueger** [comment]
Yes the Cosync service is still being maintained. We got sidetracked this last month working on a self-hosted version of the CosyncJWT service. This new service will allow you to run a CosyncJWT authentication server on your own host and administer it through the Cosync Portal. We fixed the certificate issue with https://cosync.io - turned out that the associated credit card for the certification renewal had expired. I will look into the metadata issue that you brought up.

Richard Krueger
